src/crimson/osd/main.cc

```diff
@@ -238,6 +239,16 @@ int main(int argc, const char* argv[])
           local_conf().parse_config_files(conf_file_list).get();
           local_conf().parse_env().get();
           local_conf().parse_argv(config_proxy_args).get();
+          std::ofstream log_file_stream;
```

Comment:
It would be nice!

Reply:
One thing: right now there shouldn't be a difference but for the sake of being future-proof it might be desired to instantiate the resetter before chaning the stream.

Reply:
done